api_destination.tf

```diff
@@ -27,6 +27,7 @@ resource "aws_cloudwatch_event_connection" "connection" {
 resource "aws_cloudwatch_event_target" "event_api" {
   for_each = var.targets.event_api
 
+  target_id      = each.key
```

Comment:
will this allow the caller to set the id? not sure how `key` works? could i expect this:

```tf
module "quote_extracts_status_updates" {
  source  = "highwingio/event-mapping/aws"
  version = "0.6.6"

  bus_name  = "highwing"
  rule_name = "trigger-quote-extracts-status-updates"
  event_patterns = [
    "event.QuoteExtractStarted",
    "event.QuoteExtractCompleted",
    "event.QuoteExtractFailed"
  ]
  targets = {
    broker_suite = [{
      id = "bananas"
      name = "BrokerSuite"
      arn  = module.broker_suite.function_arn
    }]
  }
}
```

Reply:
Oh, i see, actually in this case the rule_id would be the "broker_suite" key?